common/defines/babylon_defines.txt

```diff
@@ -96,10 +96,10 @@ NPops = {
 
 NPolitics = {
 	BASE_IG_APPROVAL = 0				# Baseline IG approval
-	MIN_IG_APPROVAL = -25				# Minimum IG approval
-	MAX_IG_APPROVAL = 25				# Maximum IG approval
-	MIN_IG_APPROVAL_FROM_LAWS = -8		# IG cannot have approval lower than this as a total from laws 'on the book'
-	MAX_IG_APPROVAL_FROM_LAWS = 8		# IG cannot have approval higher than this as a total from laws 'on the book'
+	MIN_IG_APPROVAL = -25				#20 Minimum IG approval
+	MAX_IG_APPROVAL = 25				# 20 Maximum IG approval
```

Comment:
What is the point of this? The thresholds don't use this extra opinion, so why would 5 extra approval or -5 extra approval matter?